### PR TITLE
feat(events): added more filtering on events

### PR DESCRIPTION
**Pull request by Her0elt**
## Proposed changes
added the filter fields:
- open_for_sign_up which is used to filter events that you get sign up for used like this open_for_sign_up=true
- start_date_before, start_date_after = which is a range filter for event start dates
- organizer which can show events that one group is hosting

Issue number: closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] The fixtures have been updated if needed (for migrations)
- [ ] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [ ] Format and lint (`make format` and `make flake8`) was run locally without failures

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: app/content/filters/event.py [hunk 1]
```diff
@@ -20,3 +29,12 @@ def filter_expired(self, queryset, name, value):
         if value:
             return queryset.filter(end_date__lt=time).order_by("-start_date")
         return queryset.filter(end_date__gte=time).order_by("start_date")
+
+    def filter_open_for_sing_up(self, queryset, name, value):
```
Comment (trthingnes) [inline app/content/filters/event.py:33]: Tror du mente `open_for_sign_up` her.

**Review by trthingnes — CHANGES_REQUESTED** [review]
Fix typo, then 🥇

File: app/content/filters/event.py [hunk 2]
```diff
@@ -8,10 +12,14 @@ class EventFilter(FilterSet):
     """ Filters events by category and expired. Works with search query """
 
     expired = BooleanFilter(method="filter_expired", label="Newest")
+    start_date = DateFromToRangeFilter()
```
Comment (olros) [inline app/content/filters/event.py:15]: Er et edge-case her hvis du har et arrangement som starter rett før månedsskiftet og slutter rett etter. Da vil det ikke komme med i den ønskede måneden, selv om det burde det.

Men er et veeldig spesifikt case så spørs om det er verdt å bruke tid på å fikse med 2 filtre istedenfor bare dette ene

**Review by olros — APPROVED** [review]
yey

**Pull request merged**